### ISSUE TITLE
Rocket Knight Adventures: Re-Sparked

### ISSUE THREAD

**Issue by legolas119** [Ryujinx/Ryujinx-Games-List#4995]
### Game Name

Rocket Knight Adventures: Re-Sparked

### Game Update Version

1.0.0

### Emulator Build Version

1.1.1336

### Graphics Backend

OpenGL and Vulkan

### Hardware Specs

CPU: (AMD Ryzen 3700X)
GPU: (RTX 2060 Super)
GPU Driver version: (552.22)
RAM: (16GB)

### Game State

The game immediately crashes as soon as i start it with the emulator

### Log File

[Ryujinx_1.1.1336_2024-06-19_22-59-41.log](https://github.com/user-attachments/files/15905842/Ryujinx_1.1.1336_2024-06-19_22-59-41.log)

**Comment by gdkchan** [comment]
Might work if you enable ignore missing services, FWIW.

**Comment by legolas119** [comment]
yeah, thanks! it works! is it better to leave that option checked by default?
thanks!
